.github/workflows/main.yml

```diff
@@ -55,14 +67,6 @@ jobs:
       run: |
```

Comment:
Can you also update the python version?

Reply:
to 3.9 or use a matrix?

Reply:
> to 3.9 or use a matrix?

matrix would be better.

Reply:
@epicadk done , I've added 3 versions for now ,  not sure if it works with ` latest`

Reply:
I don't think we need 3.6

Reply:
Can you not add latest as a version in the matrix?

Reply:
@epicadk , 3.6 is still the only version being used in MS backend workflow. Shouldn't we change MS to use the matrix as well there so it's consistent with BIT? (Although BIT actually should be the one that following whatever MS has 😁). Especially since the reason behind this update is to apply `zoneinfo` module in BIT-MS integration as per your suggestion [here](https://github.com/anitab-org/mentorship-backend/pull/1074/files#r615392263).

Reply:
@mtreacy002 https://github.com/anitab-org/mentorship-backend/issues/1078